application/tests/org.openjdk.jmc.jolokia.test/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,17 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: RJMX Test
```

Comment:
Update the name to Jolokia Test or something similar

Reply:
0fc546a02a936430700483d29503dad4505d1b64